recovery.m

```diff
@@ -1,119 +1,133 @@
 function [confusion, corrs] = recovery(model, nsims, or_params, or_errors)
 
-nfits = 100; nevals = 10000;
-
-if model == "both"
-    model = ["ci", "nr"];
-    nmodels = 2;
-    bic = struct;
-    bic.ci = NaN(nsims, 2);
-    bic.nr = NaN(nsims, 2);
-else
-    nmodels = 1;
-end
-or_n = size(or_errors, 1);
-corrs = NaN(nmodels, 4);
-
-priors = [];
-likelihoods = [];
-
-for i=1:9
-    for j=1:9
-        if i+j>=4 && i+j<=16 && ~(i==5 && j==5)
-            priors = [priors; i/10];
-            likelihoods = [likelihoods; j/10];
-            if ismember(i, [3, 4, 6, 7]) || ismember(j, [3, 4, 6, 7])
-                priors = [priors; i/10];
-                likelihoods = [likelihoods; j/10];
+    nfits = 100; nevals = 10000;
```

Comment:
I don't understand what this does or why
Can parameters of the analysis be an _input_ or _configuration_ of the program, vs part of the program?

Reply:
e.g. PARAMS.m

Reply:
(disclaimer: I don't know how to matlab)